ceros/ShareJS
# Change the size of docker images

Resolves ceros/ceros-product#3944

This changes are regarding the review of the Dockerfiles in order to reduce the size of the image of ShareJS

## Deploy notes:

No need to deploy.

## Testing Steps:
To test it you should deploy ShareJS in the docker environment and check to see if everything is working.
